src/common/http_header.cc

```diff
@@ -283,6 +281,10 @@ Connection::write(std::ostream& os) const {
     case ConnectionControl::KeepAlive:
         os << "Keep-Alive";
         break;
+    case ConnectionControl::Ext:
```

Comment:
When it should not happen, the you shoudl throw a execption or so. but a modern compile will complain that that value from the enum is not handled in the switch-case